FrameworkSystem/Client/MonitoringClient.py

```diff
@@ -3,6 +3,8 @@
     Uses RPC Framework/Monitoring service. Calls registerActivities exposed function
 """
 
+from __future__ import absolute_import
+from six.moves import range
```

Comment:
why ? `xrange` does the trick no ?

Reply:
Nope, I guess this was done by `python-modernize`. I will remove it.

Reply:
python3 doesn't have `xrange`

Reply:
YesI know, but `2to3` does the migration nicely